### DISCUSSION TITLE
Game textures going nuts

### DISCUSSION THREAD

**Discussion by NotCrafted** [ppy/osu#25445] [Q&A]
### Type

Game behaviour

### Bug description

Just downloaded the game and as you can see from the clip, It is broken. I tried reinstalling the game to see if something different happens but it didn't really help. I haven't tried playing any maps with it so ask me if yall need it.

Osu!Lazer version : 2023.1114.0
Android version : 12
(Pls ask me other infos if necessary I don't know which ones to add)

### Version

2023.1114.0

### Logs

[logs.zip](https://github.com/ppy/osu/files/13348964/logs.zip)

**Comment by NotCrafted** [comment]
I swear I added a link of the screen recording, might have refreshed the page and didn't checked.
Anyways here's the link https://cdn.discordapp.com/attachments/1162720950312243211/1173967205008941096/Screenrecorder-2023-11-14-19-07-37-559.mp4?ex=6565e128&is=65536c28&hm=775863f20d9a953f69862d5a7a56f98688f75d60a87ade613fb6dac78ff2a013&

**Reply by NotCrafted** [reply]
Update : After trying 2023.1114.1, The bug still persist.

**Comment by bdach** [comment]
Devices with mali GPUs have notoriously bad drivers and this honestly looks like driver level breakage. Not sure we can do much here.
